core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -33,7 +33,7 @@ import scala.concurrent.{Await, ExecutionContext}
 object ControllerConfig {
   def default: ControllerConfig =
     ControllerConfig(
-      statusUpdateIntervalMs = Option(100)
+      statusUpdateIntervalMs = Option(500)
```

Comment:
besides this default value, can we also make it read from config file so that we can customize it on different deployments?